docs/standard/exceptions/index.md

```diff
@@ -30,38 +30,37 @@ Traditionally, a language's error-handling model relied on either the language's
 
-## Common Exceptions
+## Common exceptions
 
 The following table lists some common exceptions with examples of what can cause them.
 
-| Exception type | Base type | Description | Example |
-| -------------- | --------- | ----------- | ------- |
-| <xref:System.Exception> | <xref:System.Object> | Base class for all exceptions. | None (use a derived class of this exception). |
-| <xref:System.IndexOutOfRangeException> | <xref:System.Exception> | Thrown by the runtime only when an array is indexed improperly. | Indexing an array outside its valid range: `arr[arr.Length+1]` |
-| <xref:System.NullReferenceException> | <xref:System.Exception> | Thrown by the runtime only when a null object is referenced. | `object o = null; o.ToString();` |
-| <xref:System.InvalidOperationException> | <xref:System.Exception> | Thrown by methods when in an invalid state. | Calling `Enumerator.GetNext()` after removing an Item from the underlying collection. |
-| <xref:System.ArgumentException> | <xref:System.Exception> | Base class for all argument exceptions. | None (use a derived class of this exception). |
-| <xref:System.ArgumentNullException> | <xref:System.Exception> | Thrown by methods that do not allow an argument to be null. | `String s = null; "Calculate".IndexOf (s);` |
-| <xref:System.ArgumentOutOfRangeException> | <xref:System.Exception> | Thrown by methods that verify that arguments are in a given range. | `String s = "string"; s.Substring(s.Length+1);` |
-
-## See Also
-
-* [Exception Class and Properties](exception-class-and-properties.md)
-* [How to: Use the Try-Catch Block to Catch Exceptions](how-to-use-the-try-catch-block-to-catch-exceptions.md)
-* [How to: Use Specific Exceptions in a Catch Block](how-to-use-specific-exceptions-in-a-catch-block.md)
-* [How to: Explicitly Throw Exceptions](how-to-explicitly-throw-exceptions.md)
-* [How to: Create User-Defined Exceptions](how-to-create-user-defined-exceptions.md)
-* [Using User-Filtered Exception Handlers](using-user-filtered-exception-handlers.md)
-* [How to: Use Finally Blocks](how-to-use-finally-blocks.md)
-* [Handling COM Interop Exceptions](handling-com-interop-exceptions.md)
-* [Best Practices for Exceptions](best-practices-for-exceptions.md)
-
-To learn more about how exceptions work in .NET, see [What Every Dev needs to Know About Exceptions in the Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/exceptions.md).
+| Exception type | Description | Example |
+| -------------- | ----------- | ------- |
+| <xref:System.Exception> | Base class for all exceptions. | None (use a derived class of this exception). |
+| <xref:System.IndexOutOfRangeException> | Thrown by the runtime only when an array is indexed improperly. | Indexing an array outside its valid range: <br /> `arr[arr.Length+1]` |
```

Comment:
nit: current code example is correct (example of an exceptional case), but what about `arr[arr.Length]` to clearly show that indexing is zero-based?

Reply:
I think we could keep the very obvious example. @rpetrusha @BillWagner thoughts?

Reply:
I agree,, @mairaw. There's still occasional confusion about one-based and zero-based indexing, so this makes it obvious that the index is out of bounds without requiring explanation.